examples/in-composition/composition.yaml

```diff
@@ -206,7 +206,7 @@ spec:
       readinessChecks:
         - type: None
     - base:
-        apiVersion: kubernetes.crossplane.io/v1alpha1
+        apiVersion: kubernetes.crossplane.io/v1beta1
```

Comment:
think this needs to be fixed in master branch 

`kubernetes.crossplane.io/v1alpha2`